src/main/java/net/masterthought/cucumber/json/DocString.java

```diff
@@ -17,4 +17,8 @@
     public String getValue() {
         return value;
     }
+
+    public String getValueWithBreakingSpace() {
+        return value!=null?value.replaceAll("(\r\n|\r|\n|\n\r)", "<br/>"):value;
```

Comment:
Makes sense. I still think that we should convert carriage returns with <br/> but that can be done before calling the reporting tool.